tests/php/Http/Controllers/Api/VolumeControllerTest.php

```diff
@@ -220,34 +221,44 @@ public function testCloneVolume()
     {
         $volume = $this->volume(
             ['created_at' => '2022-11-09 14:37:00',
-            'updated_at' => '2022-11-09 14:37:00',])->fresh();
+                'updated_at' => '2022-11-09 14:37:00',])->fresh();
         $project = ProjectTest::create();
 
         $this->doTestApiRoute('POST', "/api/v1/volumes/{$volume->id}/clone-to/{$project->id}");
 
         $this->be($project->creator);
-        $this->postJson("/api/v1/volumes/{$volume->id}/clone-to/{$project->id}")
+        $this->postJson("/api/v1/volumes/{$volume->id}/clone-to/{$project->id}",
+            ['volume' => $volume, 'project' => $project])
```

Comment:
The additional parameters shouldn't be required. The volume and project are fetched by their ID in the `CloneVolume` request.
```suggestion
        $this->postJson("/api/v1/volumes/{$volume->id}/clone-to/{$project->id}")
```